_posts/2017-09-14-survey-results.md

```diff
@@ -0,0 +1,218 @@
+
+## Table of contents
+
+* [About our users](#about-our-users)
+* [Pa11y CLI](#pa11y-cli)
+* [Pa11y Dashboard](#pa11y-dashboard)
+* [Pa11y Webservice](#pa11y-webservice)
+* [Pa11y CI](#pa11y-ci)
+* [What we plan to improve](#what-we-plan-to-improve)
+	* [Documentation, tutorials, and alternatives](#documentation-tutorials-and-alternatives)
+	* [Code changes](#code-changes)
+* [Conclusions](#conclusions)
+
+
+## About our users
+
+More than three quarters of our respondents are developers. 
+
+![Pie chart: Do you mainly see yourself as a... Developer 72%; QA tester 5%; Accessibility tester 6%; UX specialist 6%; Other 11%](asset/image/2017-survey/1-job-type.png)
+
+Over three quarters of our respondents use Pa11y for work. 
+
+20% of respondents use Pa11y only for personal projects, and 10% for both work and personal. There's no particular pattern in the way our respondents use Pa11y and the way they describe their job roles. 
+ 
+![Bar chart: What do you use Pa11y for?](asset/image/2017-survey/1-use-case.png)
+
+
+## Pa11y CLI
+
+The Pa11y command line tool is the “core” tool. It’s a simple command line interface that loads a URL and an optional configuration file, then runs HTML Codesniffer to generate a report on any accessibility errors it finds.  
+
+Two thirds of respondents have used the Pa11y CLI. 
+
+![Pie chart: Have you ever used the Pa11y command line tool? Used 67%; Not used 33%](asset/image/2017-survey/2-pa11y-cli.png)
+
+Of those who _haven’t_ used it, one third of respondents say that it’s too difficult to set up, and one third have simply never heard of it. The rest just don't have a need for it. 
+
+![Bar chart: Is there a reason that you've never used the Pa11y command line tool?](asset/image/2017-survey/2-not-used.png)
+
+Of those who _have_ used Pa11y CLI, we asked how easy/hard it is to set up. None of these respondents found it very difficult, but otherwise there is an even spread across the easy-to-hard spectrum. Most of the respondents who have used Pa11y CLI are developers; we’d expect this for a command-line tool. Greater familiarity with the command line amongst this group will also skew the results towards the “easier” end of the spectrum. 
+
+![Bar chart, scale of 1 (easier) to 5 (harder): How easy did you find it to set up the Pa11y CLI? 25% each for 1 to 4](asset/image/2017-survey/2-setup.png)
+
+Once set up, most people find Pa11y CLI easy to use. Those that found it more difficult tend to be doing more complex things with it, and overall people find it more difficult to integrate Pa11y CLI into their existing workflow. 
+
+![Bar chart, scale of 1 (easier) to 5 (harder): How easy did you find it to use the Pa11y CLI? 30% for 1; 25% for 2; 15% for 3; 25% for 4](asset/image/2017-survey/2-use.png)
+![Bar chart, scale of 1 (easier) to 5 (harder): How easy did you find it to integrate the Pa11y CLI with your workflow? 50% for 1; almost 15% for 2 and 3; 25% for 4](asset/image/2017-survey/2-integrate.png)
```

Comment:
These two bar charts appear to show the same data but with different titles. The second image doesn't match the values you give in your _excellent_ `alt` text

Reply:
oh, it's actually the alt text that's wrong - the data is the same for both charts. I'll see what I can do to combine them into one chart like I did with the webservice one